source/_posts/2023-04-27-year-of-the-voice-chapter-2.markdown

```diff
@@ -0,0 +1,158 @@
+---
+layout: post
+title: "Year of the Voice - Chapter 2: Let's talk"
+description: "Talk to your smart home and let it talk back with our new voice assistant features."
+date: 2023-04-27 00:00:00
+date_formatted: "April 27, 2023"
+author: Paulus Schoutsen
+comments: true
+categories: Assist
+og_image: /images/blog/2023-04-27-year-of-the-voice-chapter-2/social.png
+---
+
+
+This year is Home Assistant’s [Year of the Voice]. It is our goal for 2023 to let users control Home Assistant in their own language. Today we’re presenting Chapter 2, our second milestone building towards this goal.
+
+In [Chapter 1] we focused on intents – what the user wants to do. As of today, the Home Assistant community has translated common smart home commands and responses into [45 languages], closing in on the 62 languages supported by Home Assistant.
+
+For Chapter 2, we’ve expanded beyond text to now include audio; specifically, turning audio (speech) into text, and text back into speech. With this functionality, Home Assistant’s Assist feature is now able to provide a full voice interface for users to interact with.
+
+A voice assistant also needs hardware, so today we’re launching ESPHome support for Assist and; to top it off: we’re launching the World’s Most Private Voice Assistant. Keep reading to see what that entails.
+
+_To watch the video presentation of this blog post, including live demos, check [the recording of our live stream.][live-stream]_
+
+<lite-youtube videoid="0YJzLIMrnGk" videotitle="World’s Most Private Voice Assistant"></lite-youtube>
+
+[Year of the Voice]: https://www.home-assistant.io/blog/2022/12/20/year-of-voice/
+[Chapter 1]: https://www.home-assistant.io/blog/2023/01/26/year-of-the-voice-chapter-1/
+[45 languages]: https://home-assistant.github.io/intents/
+[live-stream]: https://youtube.com/live/Tk-pnm7FY7c?feature=share
+
+<!--more-->
+
+## Composing Voice Assistants
+
+The new [Assist Pipeline integration] allows you to configure all components that make up a voice assistant in a single place.
+
+For voice commands, pipelines start with audio. A speech-to-text system determines the words that were spoken by the user, which is then forwarded to a conversation agent. The intent is extracted from the text by the agent and executed by Home Assistant. At this point, “turn on the light” would cause your light to turn on 💡. The last part of the pipeline is text to speech, where the agent’s response is spoken back to you. This may be a simple confirmation (“Turned on light”) or the answer to a question, such as “which lights are on?”
```

Comment:
```suggestion
For voice commands, pipelines start with audio. A speech-to-text system determines the words the user speaks, which are then forwarded to a conversation agent. The intent is extracted from the text by the agent and executed by Home Assistant. At this point, “turn on the light” would cause your light to turn on 💡. The last part of the pipeline is text-to-speech, where the agent’s response is spoken back to you. This may be a simple confirmation (“Turned on light”) or the answer to a question, such as “Which lights are on?”
```